source/extensions/path/pattern_template_lib/proto/pattern_template_rewrite_segments.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package envoy.extensions.pattern_template;
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.pattern_template";
+option java_outer_classname = "PatternTemplateRewrite";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/pattern_template/pattern_template";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Pattern Template Rewrite]
```

Comment:
perhaps im missing something - but to be used by protodoc this needs to be in the api

Reply:
It is internal to the extension. No need to have user facing docs

Reply:
in which case we dont need the protodoc annotations

Reply:
Ok will remove